src/System.Runtime.WindowsRuntime/tests/System.Runtime.WindowsRuntime.Tests.csproj

```diff
@@ -1,23 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I realize these tests currently only have a uap configuration, but that seems wrong. I would expect we should be able to run at least some of these tests outside of an app container.

Reply:
Yes, these tests should stay and be enabled on netcoreapp Windows (where WinRT is supported).